notebooks/multimodal_vdj/01_preprocessing.ipynb

```diff
@@ -0,0 +1,3169 @@
+{
```

Comment:
<ol><li>Eventhough -&gt; Even though</li><li>therefor -&gt; therefore (therefor is mostly slang)</li><li>There's a TODO here :)</li><li>So what do we do with unproductive AIRs? -&gt; Remove? Explain what consequences this has, please. (cleaner data, or losing information, ...)</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>